bcgov/HOUS-permit-portal
# Feature/bphh 881/per role eula

## Description

Different EULAs for different roles.
add a EULA updater and some starting EULAs (Skye to provide better content html later)

bonus, fix spinner on logging out.

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-881

## Steps to QA

Run the migration and the  EulaUpdater with `rails db:migrate`
if you need to update the eulas, updtate the HTMLs in the eula folder and run

`rake db:update_eulas`

log in as submitter and reviewer and confirm that the correct eula is presented
log in as super admin and confirm nothing is presented
